packages/web/src/common/store/lineup/sagas.js

```diff
@@ -39,7 +39,7 @@ const IS_NATIVE_MOBILE = process.env.REACT_APP_NATIVE_MOBILE
 // This is copied from web/src/utils since moving it to common would require
 // changing 90+ files. This will be ignored in RN-reloaded, since we check
 // isNativeMobile from storeContext first.
-const isMobile = () => {
+export const isMobile = () => {
```

Comment:
does `IS_NATIVE_MOBILE` have any meaning in common since we pull process env? I think we should just remove that part of this check entirely since this util really just checks user agent stuff

Reply:
i think we put that there because this function is only meant to be run in web - in the two cases that we use it (this file and the search results track lineup sagas), we check isNativeMobile from storeContext first -- so effectively this does not run if there is no process.env.

Reply:
cc @dylanjeffers

Reply:
yeah its a good callout! imo we are gonna do an IS_NATIVE_MOBILE round soon so can probably get it there, cause like @nicoback2  is saying it's not gonna get hit anyway. fwiw there are lot of cases like this littered around the app still.